lib/models/App.js

```diff
@@ -335,6 +336,44 @@ class App {
     return this
   }
 
+  loadPlugins() {
+    if (!this.config) {
```

Comment:
can let `loadConfig` check this.config itself

Reply:
a bit different. this part of code means to ensure config is ready in these phases. Maybe we can move the this.config to a lazy loading method. How is it :)